portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsSingleSelectionField.testcase

```diff
@@ -153,13 +153,58 @@ definition {
 	}
 
 	@description = "Verify that options can be added"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test OptionsCanBeAdded {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-132205 OptionsCanBeAdded pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Single Selection");
+
+		Form.save();
+
+		Click(
+				key_fieldName = "Single Selection",
+				locator1 = "Form#FIELD_LABEL");
+
+		Type(
+			key_rowNumber = "1",
+			locator1 = "FormFields#OPTIONS_OPTION_VALUE_FIELD",
+			value1 = "Option 1");
+
+		Type(
+			key_rowNumber = "2",
+			locator1 = "FormFields#OPTIONS_OPTION_VALUE_FIELD",
+			value1 = "Option 2");
+
+		Form.save();
+
+		FormsAdminNavigator.gotoBack();
+
+		FormsAdminNavigator.gotoEditFormByName(formName = "Form 1");
+
+		AssertElementPresent(
+			key_optionLabel = "Option 1",
+			key_position = "1",
+			locator1 = "FormFields#SELECT_ASSERT_RADIO_OPTION_ORDER");
+
+		AssertElementPresent(
+			key_optionLabel = "Option 2",
+			key_position = "2",
+			locator1 = "FormFields#SELECT_ASSERT_RADIO_OPTION_ORDER");
```

Comment:
Hi Paulo, the path that you have created here looks great, may I ask you to create a macro using it?

I think we can name it viewSingleSelectionOption and you can create it inside [FormFields.macro](https://github.com/liferay/liferay-portal/blob/master/portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro) file.

Basically, it would take the option label and position as parameter and it will assert the option is rendered on screen.

Let me know if you need any help to create it, I'll be always available on Slack.